### DISCUSSION TITLE
is it necessary to integrate crictl  some function  into nerdctl to manage pod?

### DISCUSSION THREAD

**Discussion by ningmingxiao** [containerd/nerdctl#876] [Q&A]

**Comment by AkihiroSuda** [comment]
What do you mean?

**Reply by ningmingxiao** [reply]
I think nerdctl can use cri interface to manage pods, nerdctl can manage pods when use k8s.

**Reply by AkihiroSuda** [reply]
There's a proposal to implement containerd-native pod (sandbox) objects
https://github.com/containerd/containerd/pull/5396

We will be able to have `nerdctl sandbox` command to manage these pod/sandbox objects
